python/requirements/ml/tune-test-requirements.txt

```diff
@@ -2,16 +2,21 @@
 
 aim==3.17.5
 
-gpy==1.10.0
+gpy==1.10.0; python_version <= '3.9'
```

Comment:
what version of `gpy` will be used for other python versions?

Reply:
the latest version which is 0.3.5 will break so I'll upgrade it for python 3.11 in another PR